Docs update (#260)
NCAR/geocat-comp: docs/getting-started.rst

.. currentmodule:: geocat.comp

.. _getting-started:

Getting Started
===============
This is a quick guide to getting set up and familiar with GeoCAT-comp. You'll get an overview of what geocat-comp is,

> why we made it, and what functionalities it provides. You'll also get quick installation instructions, some simple
> examples, and info for where to get additional help.

Reviewer: ```suggestion
why we made it, and what functionalities it provides. You'll also get quick installation instructions and some simple
examples to get you started.
```
Removing the comment about additional help, since the support page is not part of the quick start guide anymore

Author: Yup, good catch!
